fhir-server-webapp/src/main/liberty/config/configDropins/disabled/jvm.options

```diff
@@ -1,8 +1,9 @@
-# Set a fixed max heap size
--Xmx3072M
+# ----------------------------------------------------------------------------
+# (C) Copyright IBM Corp. 2021
+# 
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
 
-# Turn on verbose classloading
-#-verbose:class
-
-# Turn on verbose garbage collection
-#-verbose:gc
+# https://www.eclipse.org/openj9/docs/xxinitialrampercentage/
+-XX:InitialRAMPercentage=50.00
+-XX:MaxRAMPercentage=90.00
```

Comment:
```suggestion
-XX:MaxRAMPercentage=80.00
```
I suggest higher than the default. We're not running parallel processes, even the heapdump is lightweight.

Reply:
totally good with higher-than-default

Reply:
maybe even 85% ?